software/wmla120_ansible/validate_firewall_setup.yml

```diff
@@ -26,26 +26,26 @@
     msg: "{{ host_firewall_status.stdout.find('http') }}"
 
 - name: Print success status of Host firewall
-  command: echo "Firewall Exists"
+  command: echo "Firewall is enabled, proceeding to open ports"
   when: host_firewall_status.stdout.find('http') != -1
 
 - name: Else print fail status
-  command: echo "Firewall Does Not Exist"
+  command: echo "WARNING - Firewall Not Enabled"
   when: host_firewall_status.stdout.find('http') == -1
 
-- name: Open Ports for Spectrum Conductor
+- name: Open Ports for Spectrum Conductor - if firewall is enabled
   firewalld:
     port: "{{ item }}"
     permanent: true
     state: enabled
     immediate: true
   with_items:
-     - "{{ port_9243 }}"
-     - "{{ port_9080 }}"
-     - "{{ port_5000 }}"
-     - "{{ port_5001 }}"
-     - "{{ port_27017 }}"
-     - "{{ port_6379 }}"
+    - "{{ ports.DLI_DLPD_REST_PORT }}/tcp"
+    - "{{ ports.DLI_DLPD_REST_PORT_SSL_NOT_ENABLED }}/tcp"
+    - "{{ ports.DLI_INSIGHTS_MONITOR_PORT }}/tcp"
+    - "{{ ports.DLI_INSIGHTS_OPTIMIZER_PORT }}/tcp"
+    - "{{ ports.DLI_MONGODB_PORT }}/tcp"
+    - "{{ ports.DLI_REDIS_PORT }}/tcp"
   when: host_firewall_status.stdout.find('http') != -1
```

Comment:
should be something like;
when: host_firewall_status.stderr != 'not running'

Reply:
host_firewall_status.stdout.find('http') != -1 allows us to check for two states:
 1) The firewall is enabled 
2) http is active and enabled
I believe http should be verified to make sure subsequent steps will have no issues with file transfers across nodes if the firewall is active.